src/Makefile.am

```diff
@@ -2465,6 +2465,7 @@ tests_testHttpRequest_LDADD = \
 	$(SSLLIB) \
 	$(KRB5LIBS) \
 	$(LIBCPPUNIT_LIBS) \
+	$(SYSTEMD_LIBS) \
```

Comment:
I am curious how you picked targets that need SYSTEMD_LIBS.

I see that SYSTEMD_LIBS was added in commit 6fa8c6643 along with modification of client_side.cc to call sd_notify(). That library call is still there and was not copied to other source files. I am guessing that you are adding SYSTEMD_LIBS because of that sd_notify() call in client_side.cc. However, this PR:

* does not add SYSTEMD_LIBS to some of the targets that do include client_side.cc (e.g. tests_test_http_range);
* adds SYSTEMD_LIBS to some of the targets that do not include client_side.cc (e.g. tests_testEventLoop).

If current PR additions were done by blind trial and error, then please double check that tests_testEventLoop addition is really needed in your environment (if yes, it would be interesting to understand why it is needed). And you might want to cover tests_test_http_range, just in case and for consistency sake (especially if it turns out that 
tests_testEventLoop is _not_ needed).

----

FWIW, here is a list of all targets that contain client_side.cc AFAICT, obtained via `egrep -h '_SOURCES =|[^_]client_side.cc' src/Makefile.am | fgrep -B1 client_side.cc`.


```
squid_SOURCES = \
        client_side.cc \

tests_test_http_range_SOURCES = \
        client_side.cc \

tests_testHttpRequest_SOURCES = \
        client_side.cc \

tests_testCacheManager_SOURCES = \
        client_side.cc \

tests_testEvent_SOURCES = \
        client_side.cc \
```

Reply:
Of course, that's correct :(. In tests_testEventLoop, we dont need to add SYSTEMD_LIBS (in v5, it is needed) and for tests_test_http_range  - SYSTEMD_LIBS are needed as you mentioned above - because sd_notify() call in client_side.cc. 

I somehow removed SYSTEMD_LIBS addition from tests_test_http_range by mistake - such build would fail as well. Adding this change to v5, since client_side.o is also used in tests_test_http_range.